### DISCUSSION TITLE
[Feature] Leveraging Live Activities and Dynamic Island

### DISCUSSION THREAD

**Discussion by toorux** [immich-app/immich#12969] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have a suggestion: I hope the iOS client can utilize the new feature "Live Activities" for background synchronization and adapt it for the Dynamic Island, instead of using regular push notifications. This way, it won't repeatedly pop up banner notifications (unless the user has disabled them).

I know that others have suggested using Live Activities, but I want to emphasize that the current notification method triggers a banner for every photo backed up, which can be quite bothersome.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#8914
